gateway/sds_gateway/users/views.py

```diff
@@ -1858,34 +1863,21 @@ def _handle_file_uploads(self, request, files, relative_paths):
                     status.HTTP_200_OK,
                     status.HTTP_201_CREATED,
                 ):
-                    resp_data = response.data
-                    saved_files.append(
-                        {
-                            "name": filename,
-                            "directory": directory,
-                            "size": file_size,
-                            "uuid": (
-                                resp_data.get("uuid")
-                                if isinstance(resp_data, dict)
-                                else None
-                            ),
-                            "status": response.status_code,
-                        }
-                    )
+                    saved_files.append({"uploaded": True})
```

Comment:
It seems you're not tracking the file UUIDs anymore, which is fine _if you're not using this information_. But then, maybe `saved_files` can just be a `num_saved_files` counter instead of a list?